ansible/roles/generate-permission-files/tasks/main.yml

```diff
@@ -94,21 +78,7 @@
        - PyYAML
     delegate_to: 127.0.0.1
     run_once: yes
-    tags:
-      - configuration
```

Comment:
We do not need to install gspread, although it is not a part of change so I can't comment on that line. line`92` and change line no`76`. why do we need oauth2client as well?